cmake/CPM.cmake

```diff
@@ -251,7 +251,14 @@ function(CPMFindPackage)
     endif()
   endif()
 
-  if(CPM_DOWNLOAD_ALL)
+  set(download_only ${CPM_DOWNLOAD_ALL})
```

Comment:
the convention for local vars is `camelCase`. Also tne name "download only" can be confused with the case where a package is downloaded but not configured. Why not use something like `downloadPackage`?

Reply:
done